src/main/resources/org/springframework/integration/config/xml/spring-integration-kafka-1.2.xsd

```diff
@@ -105,6 +105,30 @@
                                             ]]></xsd:documentation>
 										</xsd:annotation>
 									</xsd:attribute>
+									<xsd:attribute name="key-class-type" use="optional" type="xsd:string" default="[B">
+										<xsd:annotation>
+											<xsd:appinfo>
+												<xsd:documentation>
+													Class type used for the value
+												</xsd:documentation>
+												<tool:annotation kind="direct">
+													<tool:expected-type type="java.lang.Class"/>
+												</tool:annotation>
+											</xsd:appinfo>
+										</xsd:annotation>
+									</xsd:attribute>
+									<xsd:attribute name="value-class-type" use="optional" type="xsd:string" default="[B">
+										<xsd:annotation>
+											<xsd:appinfo>
+												<xsd:documentation>
+													Class type used for the value
+												</xsd:documentation>
+												<tool:annotation kind="direct">
+													<tool:expected-type type="java.lang.Class"/>
+												</tool:annotation>
+											</xsd:appinfo>
+										</xsd:annotation>
+									</xsd:attribute>
 									<xsd:attribute name="value-encoder" use="optional" type="xsd:string">
```

Comment:
Generally speaking, I've tried to preserve backwards compatibility with regards to the schema, unless it was clearly impossible to do so, or it would have made things confusing/dangerous (e.g. `async`, or switching batch size from number of messages to bytes), and it seemed like in this case there wouldn't be a lot of extra effort to do so, but we can just drop support for that, too.